### PR TITLE
Updated requirements

### PR DESCRIPTION
**Pull request by clintonb**

### CODE REVIEW

**Comment by clintonb** [comment]
FYI @edx/ecommerce

File: requirements/test.txt [hunk 1]
```diff
@@ -12,5 +12,5 @@ mock-django==0.6.9
 nose-ignore-docstring==0.2
 pep8==1.6.2
 pylint==1.4.4
-selenium>=2.47.1
-testfixtures==4.2.0
+selenium>=2.48.0
```
Comment (rlucioni) [inline requirements/test.txt:15]: Have you tried running the acceptance tests after installing this updated version?

File: requirements/base.txt [hunk 2]
```diff
@@ -4,9 +4,9 @@ Django==1.8.6
 django-appconf==0.6
 django-compressor==1.5
 django_extensions==1.5.5
-django-libsass==0.3
-django-oscar==1.1
-django_simple_history==1.6.1
+django-libsass==0.5
+django-oscar==1.1.1
```
Comment (rlucioni) [inline requirements/base.txt:8]: For anyone wondering, this is a bug fix release ([release notes](https://django-oscar.readthedocs.org/en/releases-1.1/releases/v1.1.1.html)).

File: requirements/test.txt (hunk 1 above)
Comment (clintonb) [inline requirements/test.txt:15]: Yes.

File: acceptance_tests/config.py [hunk 3]
```diff
@@ -38,7 +38,7 @@ def str2bool(s):
 # MARKETING CONFIGURATION
 ENABLE_MARKETING_SITE = str2bool(os.environ.get('ENABLE_MARKETING_SITE', False))
 
-MARKETING_URL_ROOT = os.environ.get('MARKETING_URL_ROOT').strip('/')
+MARKETING_URL_ROOT = os.environ.get('MARKETING_URL_ROOT').strip('/') if ENABLE_MARKETING_SITE else None
```
Comment (rlucioni) [inline acceptance_tests/config.py:41]: Nice catch.

**Comment by rlucioni** [comment]
:+1:

**Comment by vkaracic** [comment]
Out of curiosity, do you guys use something like http://www.pypi-notifier.org/ to track package updates?

**Comment by clintonb** [comment]
We've discussed it, but haven't chosen a single tool.

## 

Clinton Blackburn

**Pull request merged**